styles.css

```diff
@@ -0,0 +1,136 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+
+.container {
+  position: relative;
+  overflow: hidden;
+  height: 100vh;
+  width: 100%;
+}
+
+.container .slider {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 500%;
+  animation: animate 15s ease-in-out infinite;
+}
+
+.container .slider .slide {
+  position: relative;
+  width: 20%;
+  height: 100%;
+  float: left;
+}
+
+.container .slider .slide .caption {
+  position: absolute;
+  bottom: 10px;
+  left: 10px;
+  right: 10px;
+  padding: 20px;
+  box-sizing: border-box;
+  color: #fff;
+  background-color: rgba(0, 0, 0, 0.8);
+  transition: 0.5s;
+}
+
+.container .slider .slide .caption h2 {
+  font-size: 34px;
+  padding: 5px;
+  transition: 0.5s;
+}
+
+.container .slider .slide .caption p {
+  font-size: 18px;
+  transition: 0.5s;
+}
+
+.container .slider .slide.slide1 {
+  background-image: url(https://www.motortrend.com/uploads/sites/5/2020/08/2021-Nissan-GT-R-NISMO-front-three-quarter-motion.jpg?fit=around%7C1000:625);
+  background-size: cover;
```

Comment:
Using links to add background images is not a good idea as the links might be removed in the future. For now its alright

Reply:
Okay, Will always use local media next time onwards